frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/StartPageView.tsx

```diff
@@ -72,52 +97,150 @@ export const StartPageView = () => {
   }, [form?.startPage, startPageFromStore])
 
   // Color theme options and other design stuff, identical to public form
-  const { titleColor, titleBg, estTimeString, colorScheme } =
-    useFormHeader(startPage)
+  const { titleColor, titleBg, estTimeString } = useFormHeader({
+    startPage,
+    hover: hoverStartPage,
+  })
 
   const { hasLogo, logoImgSrc, logoImgAlt } = useFormBannerLogo({
     logoBucketUrl,
     logo: startPage?.logo,
     agency: form?.admin.agency,
   })
 
+  const content = useMemo(() => startPage?.paragraph, [startPage?.paragraph])
```

Comment:
/nit `useMemo()` is not needed for cases like `useMemo(() => a, [a])`

```suggestion
  const content = startPage?.paragraph
```